tests/unit_tests/CMakeLists.txt

```diff
@@ -44,7 +44,7 @@ set(blas_TEST_LIST
         blas_batch
         blas_extensions)
 
-set(blas_TEST_LINK ${CBLAS_LINK})
+set(blas_TEST_LINK "")
```

Comment:
Do we still need this line?

Reply:
No, we don't, but it would improve readability as we don't need to worry if `blas_TEST_LINK` is defined somewhere else. In addition, it is consistent with the `rng` domain:
https://github.com/oneapi-src/oneMKL/blob/15946f83fc25ba130f73a65bcf3174639fb8c1a0/tests/unit_tests/CMakeLists.txt#L55-L60